YIF.Core.Domain/Repositories/InstitutionOfEducationModeratorRepository.cs

```diff
@@ -70,6 +70,17 @@ public Task<InstitutionOfEducationModeratorDTO> Get(string id)
             throw new NotImplementedException();
         }
 
+        public async Task<InstitutionOfEducationModeratorDTO> GetByUserId(string userId)
+        {
+            var moderator = await _dbContext.InstitutionOfEducationModerators.Include(m => m.Admin).AsNoTracking().FirstAsync(a => a.UserId == userId);
+            return _mapper.Map<InstitutionOfEducationModeratorDTO>(moderator);
+        }
+
+        public Task<IEnumerable<InstitutionOfEducationModeratorDTO>> GetAllUniModerators()
```

Comment:
why this method was added to interface if it not implemented??

Reply:
It was added during rebase. I decided to leave it, because somebody, who added it, can implement it later.

Reply:
I`m sorry, but this method was added by you in IInstitutionOfEducationModerators

Reply:
... Ok, I will erase it.